ipet/parsing/ReaderManager.py

```diff
@@ -296,46 +302,71 @@ def collectData(self):
         """
         assert(self.testrun != None)
 
-        # sort the files by context
-        self.filestrings = sorted(self.filestrings, key = lambda x:self.sortingKeyContext(self.filenameGetContext(x)))
+        # sort the files by context, for example: outfiles should be read before solufiles
+        if not self.testrun.inputfromstdin:
```

Comment:
that'll not make it easier finding out the solver, or do you have a good idea about that?